.github/workflows/flake8.yml

```diff
@@ -23,8 +23,5 @@ jobs:
         python -m pip install --upgrade pip wheel
         pip install -r requirements-dev.txt
 
-    - name: Run flake8
-      run: |
-        echo "::add-matcher::.github/matchers/flake8.json"
-        pre-commit run flake8 --all-files
-        echo "::remove-matcher owner=flake8::"
```

Comment:
Why remove the matcher? That makes lint issue show up on GitHub.

Reply:
See the commit message above.  `ruff --format github` makes this _much_ easier.